patches/server/1052-Add-method-to-reset-Lodestone-compasses-back-to-norm.patch

```diff
@@ -0,0 +1,45 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: DerEchtePilz <81232921+DerEchtePilz@users.noreply.github.com>
+Date: Tue, 20 Aug 2024 12:38:55 +0200
+Subject: [PATCH] Add method to reset Lodestone compasses back to normal ones
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaCompass.java b/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaCompass.java
+index 6362df65424e53098701b8d54c74b5905648b78a..71816107633baadecea28ba20046e4934f48a514 100644
+--- a/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaCompass.java
++++ b/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaCompass.java
+@@ -94,6 +94,12 @@ public class CraftMetaCompass extends CraftMetaItem implements CompassMeta {
+         if (target.isPresent() || this.hasLodestoneTracked()) {
+             tag.put(CraftMetaCompass.LODESTONE_TARGET, new LodestoneTracker(target, this.tracked));
+         }
++
++        // Paper start  - Add a method to reset Lodestone compasses back to normal ones
++        if (tracked == null) {
++            tag.builder.remove(LODESTONE_TARGET.TYPE);
++        }
```

Comment:
This shouldn't be needed, the patch is recreated from scratch each time the meta is applied (non empty ones).